keystore/v2/keystore/hmac.go

```diff
@@ -84,6 +84,25 @@ func (s *ServerKeyStore) DestroyHmacSecretKey(clientID []byte) error {
 	return nil
 }
 
+// DestroyRotatedHmacSecretKey destroy created rotated hmac symmetric key
+func (s *ServerKeyStore) DestroyRotatedHmacSecretKey(clientID []byte, index int) error {
+	log := s.log.WithField("clientID", clientID)
+	ring, err := s.OpenKeyRingRW(s.clientHMACKeyPath(clientID))
+	if err != nil {
+		log.WithError(err).Debug("Failed to open HMAC key ring for client")
+		return err
+	}
+	// Index represent virtual index of key
+	// 1 is always index of current key of the keystore
```

Comment:
this comment confuse a bit because we decrement by one below... IMHO, we should describe here, that keyrings have internal index starting from 0 and we get index here that represents virtual index starting from 1.

p.s. does it work correctly with destroyed keys? What key will be destroyed, if we have `current` + `destroyed` + `rotated` + `rotated` and we call `acra-keys destroy --index 2`? will it destroy first rotated or destroy second time already destroyed? can we test that case?